README.md

```diff
@@ -22,3 +13,11 @@ starttls-policy-cli --update-only [--policy-dir /path/to/dir]
 
 We currently only support Postfix, but contributions are welcome!
 
+## Development
+
+We recommend using `virtualenv` and `pip` to install and run `starttls-policy-cli` while developing. To get set up:
+```
+virtualenv --no-site-packages --setuptools venv --python python3.6
```

Comment:
Is there a prospect that people won't know that this will interfere with any other venv that's also called `venv`? Would it be helpful to people who aren't thinking about this if we said `starttls_venv` instead of `venv` or something?